### PR TITLE
Alerting docs: Contact points updates

### PR DESCRIPTION
**Pull request by brendamuir**
updates existing contact points docs with new redesign features
adds a new manage contact points section
separates out integrations docs ready to add more

### CODE REVIEW

File: docs/sources/alerting/alerting-rules/manage-contact-points/_index.md [hunk 1]
```diff
@@ -35,34 +35,34 @@ Complete the following steps to add a contact point.
...
 
 1. In the left-side menu, click **Alerts & IRM** and then **Alerting**.
 1. Click **Contact points** to view a list of existing contact points.
-1. Find the contact point to delete, and then click **Delete** (trash icon).
+1. On the **Contact Points** tab, find the contact point you want to delete, and then click **More** -> **Delete**.
 1. In the confirmation dialog, click **Yes, delete**.
 
 {{% admonition type="note" %}}
-You cannot delete contact points that are in use by a notification policy. You will have to either delete the notification policy or update it to use another contact point.
+You cannot delete contact points that are in use by a notification policy. YEither delete the notification policy or update it to use another contact point.
```
Comment (gillesdemey) [inline docs/sources/alerting/alerting-rules/manage-contact-points/_index.md:65]: Looks like a typo here :)

File: docs/sources/alerting/manage-notifications/view-alert-rules.md [hunk 2]
```diff
@@ -40,8 +40,8 @@ When managing large volumes of alerts, you can use extended alert rule search ca
 
 To view alerting details:
 
-1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page. By default, the List view displays.
-1. In **View as**, toggle between Grouped or State views by clicking the relevant option. See [Grouped view](#grouped-view) and [State view](#state-view) for more information.
+1. Click **Alerts & IRM** -> **Alert rules**. By default, the List view displays.
```
Comment (gillesdemey) [inline docs/sources/alerting/manage-notifications/view-alert-rules.md:43]: I _think_ the title of this section changes to just "alerting" if you don't have either Incident or OnCall installed, do we care about that?

**Review by gillesdemey — APPROVED** [review]
LGTM, just a typo and a question :)

File: docs/sources/alerting/manage-notifications/view-alert-rules.md (hunk 2 above)
Comment (brendamuir) [reply docs/sources/alerting/manage-notifications/view-alert-rules.md:43]: oh does it? hmm let me think

**Pull request merged**